src/Plugin/CommerceDashboardItem/Dummy.php

```diff
@@ -0,0 +1,42 @@
+<?php
+
+namespace Drupal\acm\Plugin\CommerceDashboardItem;
+
+use Drupal\acm\CommerceDashboardItemBase;
+use Symfony\Component\DependencyInjection\ContainerInterface;
+
+/**
+ * Class ConnectorAccessiblityRequirement.
+ *
+ * @CommerceDashboardItem(
+ *   id = "dummy_one",
+ *   title = @Translation("Dummy One - weight '1'"),
+ *   weight = 1,
+ *   group = "line",
+ * )
+ */
+class Dummy extends CommerceDashboardItemBase {
+
+  /**
+   * {@inheritdoc}
+   */
+  public static function create(ContainerInterface $container, array $configuration, $plugin_id, $plugin_definition) {
```

Comment:
Need to implement the **ContainerFactoryPluginInterface**

Reply:
CommerceDashboardItemBase does implement that interface :)